external-crates/move/crates/move-bytecode-verifier/src/reference_safety/abstract_state.rs

```diff
@@ -801,3 +907,40 @@ impl AbstractDomain for AbstractState {
         }
     }
 }
+
+fn charge_graph_size(size: usize, meter: &mut (impl Meter + ?Sized)) -> PartialVMResult<()> {
+    let size = max(size, 1);
```

Comment:
The smallest of nits but some places you use `std::cmp::max` and here you use `max`. Don't have a preference for either one, but probably best to be consistent.